<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1022</width>
    <height>600</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Flare Game Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="accessibleName">
       <string/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Main">
       <attribute name="title">
        <string>Main</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Mod"/>
    <addaction name="actionOpen_Mod"/>
    <addaction name="actionSave_Mod"/>
    <addaction name="actionClose_Mod"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Mod">
   <property name="text">
    <string>New Mod</string>
   </property>
  </action>
  <action name="actionOpen_Mod">
   <property name="text">
    <string>Open Mod</string>
   </property>
  </action>
  <action name="actionSave_Mod">
   <property name="text">
    <string>Save Mod</string>
   </property>
  </action>
  <action name="actionClose_Mod">
   <property name="text">
    <string>Close Mod</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAdd_NPC">
   <property name="text">
    <string>Add NPC</string>
   </property>
  </action>
  <action name="actionAdd_Enemy">
   <property name="text">
    <string>Add Enemy</string>
   </property>
  </action>
  <action name="actionAdd_Power">
   <property name="text">
    <string>Add Power</string>
   </property>
  </action>
  <action name="actionAdd_Item">
   <property name="text">
    <string>Add Item</string>
   </property>
  </action>
  <action name="actionCreate_Map">
   <property name="text">
    <string>Create Map</string>
   </property>
  </action>
  <action name="actionOpen_Map">
   <property name="text">
    <string>Open Map</string>
   </property>
  </action>
  <action name="actionAdd_Quest">
   <property name="text">
    <string>Add Quest</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLoad_Game">
   <property name="text">
    <string>Load Game</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionSettings_Menu">
   <property name="text">
    <string>Settings Menu</string>
   </property>
  </action>
  <action name="actionInventory">
   <property name="text">
    <string>Inventory</string>
   </property>
  </action>
  <action name="actionCharacter">
   <property name="text">
    <string>Character</string>
   </property>
  </action>
  <action name="actionPowers">
   <property name="text">
    <string>Powers</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionXP_Bar">
   <property name="text">
    <string>XP Bar</string>
   </property>
  </action>
  <action name="actionMana_Bar">
   <property name="text">
    <string>Mana Bar</string>
   </property>
  </action>
  <action name="actionHP_Bar">
   <property name="text">
    <string>HP Bar</string>
   </property>
  </action>
  <action name="actionEnemy_Bar">
   <property name="text">
    <string>Enemy Bar</string>
   </property>
  </action>
  <action name="actionMiniMap">
   <property name="text">
    <string>MiniMap</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
